backend/apis/normal-docket-pdf.test.js

```diff
@@ -28,4 +34,75 @@ describe("normal docket pdf file", () => {
     expect(actual1.disposition).toBe("Dismissed (w/o prej)");
     expect(actual1.dispositionDate).toBe("Month 00, 0000");
   });
+
+  test("always adds accountSummary property as an array", () => {
+    expect(parsedText.accountSummary.length).toBe(1);
```

Comment:
I think this should be checking if array (example: `expect(Array.isArray(parsedText.accountSummary)).toBe(true);`), instead of checking the length, because that's what the test description says.

Reply:
I think this is fine - the `length` property is something that arrays have. Close enough 😄